.travis.yml

```diff
@@ -104,6 +104,23 @@ jobs:
         # Disallow spaces before pre-compiler macros                                #
         #############################################################################
         - test/hasSpaceBeforePrecompiler
+
+        #############################################################################
+        # Disallow include brackets `"` to include external library files           #
```

Comment:
Suggestion: since `"` is hard to understand, maybe we could write:
```bash
# Enforce angle brackets `<...>` for includes of external library files
```